tests/internal-api-stress-test.php

```diff
@@ -0,0 +1,123 @@
+<?php
```

Comment:
Was this intended to run in CI?

Reply:
Yes and no - I'm not sure how I would best integrate this into CI, yet.